geomesa-convert/geomesa-convert-scripting/src/main/scala/org/locationtech/geomesa/convert/scripting/ScriptingFunctionFactory.scala

```diff
@@ -0,0 +1,52 @@
+package org.locationtech.geomesa.convert.scripting
+
+import javax.script.{Invocable, ScriptContext, ScriptEngine, ScriptEngineManager}
+
+import com.google.common.io.Files
+import org.apache.commons.io.IOUtils
+import org.locationtech.geomesa.convert.Transformers.EvaluationContext
+import org.locationtech.geomesa.convert.{TransformerFn, TransformerFunctionFactory}
+
+import scala.collection.JavaConversions._
+
+class ScriptingFunctionFactory extends TransformerFunctionFactory {
```

Comment:
Is there any benefit to adding a comment describing what this file is for, or is that somehow otherwise obvious to someone already familiar with our conventions?